.gitignore

```diff
@@ -2,3 +2,4 @@ node_modules
 npm-debug.log
 dist
 coverage
+.rpt2_cache
```

Comment:
hmmm maybe `.editorconfig` would be good to add and set insert final new line. Your making the next edit harder without final new line

Reply:
Any suggestions for a good editorconfig?

Reply:
This seems to apply for the whole repo, of course you can apply different rules for each file type: https://editorconfig.org/
```
root = true

[*]
end_of_line = lf
charset = utf-8
insert_final_newline = true
ident_style = space
ident_size = 2
```